src/components/pages/Dashboard/AppCard.js

```diff
@@ -1,31 +1,90 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
-import styled from 'styled-components';
-import Form from '../../common/Form';
-import UserDescription from '../../common/UserDescription';
-import logo from '../../../images/microLogo.png';
-
-const AppCardStyle = styled.div`
-  box-shadow: 2px 2px 2px grey;
-  margin: 10px;
-  padding: 10px;
-  background: #d5f2bb;
-  box-szing: boarder-box;
-  height: 100%;
-`;
+import React, { useState, useEffect } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import { useOktaAuth } from '@okta/okta-react';
+import { useOrgQuery, useMutationToPostApp } from '../../../hooks';
+import { Form, Input, Row, Col, Button, Typography, Divider } from 'antd';
+const { Title } = Typography;
 
 function AppCard() {
-  return (
-    <AppCardStyle>
-      <img src={logo} />
-      <h1>Application Form</h1>
-      <Link to="/">
-        <div>Home</div>
-      </Link>
-      <UserDescription />
+  const { orgid } = useParams();
+  const history = useHistory();
+  const auth = useOktaAuth();
+  // grab the org name
```

Comment:
Remember to delete comments (but I understand why they're there)

Reply:
I deleted the comment, and moved up the useEffect hook, so all the org code is together, before moving onto the mutation. Instead of jumping back and forth!